client/my-sites/site-settings/main.jsx

```diff
@@ -28,15 +27,13 @@ export class SiteSettingsComponent extends Component {
 	};
 
 	static defaultProps = {
-		section: 'general'
+		section: 'guidedTransfer'
```

Comment:
This default prop is a red flag. Is it the case that this component needs to change to become `<GuidedTransfer/>`?